executor/adapter.go

```diff
@@ -110,7 +110,9 @@ type ExecStmt struct {
 	// Text represents the origin query text.
 	Text string
 
-	ctx            context.Context
+	StmtNode ast.StmtNode
+
+	Ctx            context.Context
```

Comment:
Why export it?

Reply:
It will be constructed by another package.